### DISCUSSION TITLE
Pull Request 의 제목을 한글로 적는 것에 대해 어떻게 생각하시나요?

### DISCUSSION THREAD

**Discussion by minjongbaek** [prgrms-fe-devcourse/FEDC3_DigDigDeep_Yuri#247] [General]
PR을 작성할 때 제목을 브랜치 명을 따라 영어로 적다보니 PR을 볼 때 한 눈에 들어오지 않는 것 같아요.

그래서 **브랜치 이름은 prefix/#[이슈번호]** 로 작성하고, **PR 제목은 한글로 구현사항에 대한 내용을 짧게 요약**하면 좋을 것 같다고 생각했어요.

다른 분들은 어떻게 생각하시나요?

**Comment by mxx-kor** [comment]
한글로 PR 제목 작성하는게 더 좋은 방법인거 같습니다. 그런데 Prefix/브랜치 이름/#이슈번호 이런 형태를 말하고자 하셨던거 맞나요? 브랜치 이름은 빠지면 구분하기 어려울거 같아서요

**Reply by minjongbaek** [reply]
**#이슈번호 PR제목**입니다! 이 #236 PR을 예시로 들자면,

> [#230] 프로필 페이지에 보여줄 목록이 없는 경우 예외 처리

처럼요!

**Comment by dazzel3** [comment]
PR 제목을 한글로 작성하는 것에 대해 동의합니다!

브랜치명을 예시로 작성해보면 `feat/#123` 이걸까요? 아님 `feat/#123-create-post` 이걸까요? 
저는 후자가 더 좋을 것 같다는 의견입니다. 이유는 브랜치 명에 내용이 없으면 어떤 작업을 하는 브랜치인지 이슈 번호를 찾아서 확인해야 하는 번거로움이 있지 않을까 해서요ㅜ!

**Reply by minjongbaek** [reply]
제가 말씀드린 브랜치 내용의 예는 전자입니다! `feat/#123` 처럼요!

작업 내용을 한 영문장으로 단축하기 어려운 경우가 종종 있었던 것 같아요. **지금까지 유리팀의 작업 방식을 보았을 때 브랜치로 작업내용을 추적하기 보다는 이슈로 추적**을 하고 있고, **작업하던 브랜치는 메인에 병합되는 순간 삭제를 하기 때문에** 브랜치 명에 작업 내용을 써놓는 것의 의미가 큰가 해서요.

**Reply by dazzel3** [reply]
> 작업 내용을 한 영문장으로 단축하기 어려운 경우가 종종 있었던 것 같아요.

음.. 맞아요. 그런 경우도 많았어서 `feat/#123` 이 방식으로 가는 것도 좋을 것 같습니다~
